### DISCUSSION TITLE
Time series classification with flow forecast

### DISCUSSION THREAD

**Discussion by liuliu-c** [AIStream-Peelout/flow-forecast#479] [Q&A] [answered]
Hi!

I would like to use flow forecast to do time series classification but the tutorials I found were about forecasting. Could you please provide some suggestions if we would like to implement this framework for the classification task?

Thank you!

**Comment by isaacmg** [comment, marked as answer]
Yes we do actually have a Time Series Classification tutorial https://www.kaggle.com/isaacmg/time-series-classification-with-flow-forecast-ff
Also see https://flow-forecast.atlassian.net/wiki/spaces/FF/pages/1082163201/Training+Deep+Time+Series+Classification+Anomaly+Models

**Comment by isaacmg** [comment]
It is true that TS Classification is less developed and tested so if you run into errors or have additional questions please let me know.
